main.yml

```diff
@@ -12,6 +12,8 @@ advisor:
     sub_apps:
       - id: recommendations
         title: Recommendations
+      - id: systems
```

Comment:
@kruai gotta question about https://github.com/RedHatInsights/insights-advisor-frontend/pull/653/files#diff-14b1e33d5bf5649597cdc0e4f684daddR13

we're doing `systems(redirect) { insights.chrome.appNavClick({ id: 'systems', redirect }); }` as is, without this pr merged when navigating to /advisor/systems the nav item that's selected is Compliance -> Systems, wondering if advisor needs to add any additional info to the nav click to ensure the correct selection is made

Reply:
I wouldn't think so, but you're free to [test this change locally](https://github.com/RedHatInsights/cloud-services-config#testing-your-changes-locally) to make sure it works before I approve it!

Reply:
@AllenBW what if you give this a unique id, like `advisor_systems`?

Reply:
@shdunning issue with that is url redirect is built off the id, which means we'd have to use the path `/advisor_systems` 👎 
<img width="1520" alt="Screen Shot 2020-05-27 at 12 35 54 PM" src="https://user-images.githubusercontent.com/6640236/83047663-ac613d80-a016-11ea-91c9-6aaa37088e6b.png">

Reply:
oh rite - bummer :-(

Reply:
Yeah, that's really unfortunate... @karelhala is there any special config I could have CSC provide to Chrome to mitigate this issue? Perhaps something like the `group` tag?

Reply:
https://projects.engineering.redhat.com/browse/RHCLOUD-6708 is the jira for work fixin this...